cocotb_test/simulator.py

```diff
@@ -394,6 +499,9 @@ def build_command(self):
 
 
 class Questa(Simulator):
+    has_multitop = True
+    has_namedlib = True
```

Comment:
```suggestion
    namedlib_capable = True
```
?

Reply:
This naming makes the intent more clear. Will apply it to multitop, too.